bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/ElectionKeyPair.cs

```diff
@@ -1,64 +1,12 @@
-﻿using ElectionGuard.UI.Lib.Models;
-
-namespace ElectionGuard.ElectionSetup;
-
-public class EncryptionKeyPair : DisposableBase

```

Comment:
this class didn't appear to actually add any additional functionality over the elgamal keypair it is constructed from so i removed it.